app/scripts/minmax/dimMinMax.controller.js

```diff
@@ -508,23 +508,24 @@ const _ = require('underscore');
                                 value: 0,
                                 tier: 0,
                                 name: 'Intellect',
-                                icon: 'images/Intellect.png'
+                                icon: require('app/images/Intellect.png'),
                               },
                               STAT_DISCIPLINE: {
                                 value: 0,
                                 tier: 0,
                                 name: 'Discipline',
-                                icon: 'images/Discipline.png'
+                                icon: require('app/images/Discipline.png'),
                               },
                               STAT_STRENGTH: {
                                 value: 0,
                                 tier: 0,
                                 name: 'Strength',
-                                icon: 'images/Strength.png'
+                                icon: require('app/images/Strength.png'),
```

Comment:
I'm trying to get these images working

<img width="276" alt="screen shot 2016-12-31 at 11 08 33 am" src="https://cloud.githubusercontent.com/assets/46142/21574577/80ff72a2-cf49-11e6-87f0-1f9e2b1a39c2.png">

But I don't really know where they're defined... This was my first guess, but this code doesn't appear to be executed when rendering the inventory page. `console.log` and breakpoints arent hit. Any idea where/how those images are being referenced?